verify-replace.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+TMP_DIR=/tmp/
+BASE_REPO_PATH=$(mktemp -d ${TMP_DIR}replace-verify.XXX)
+GH_BASE_URL_KS=https://github.com/kubesaw/
+GH_BASE_URL_CRT=https://github.com/codeready-toolchain/
+GH_KSCTL=${GH_BASE_URL_KS}ksctl
+GH_HOST=${GH_BASE_URL_CRT}host-operator
+GH_MEMBER=${GH_BASE_URL_CRT}member-operator
+GH_REGSVC=${GH_BASE_URL_CRT}registration-service
+GH_E2E=${GH_BASE_URL_CRT}toolchain-e2e
+GH_TC=${GH_BASE_URL_CRT}toolchain-common
```

Comment:
alternatively you could define an array:
```suggestion
declare -a repos=("${GH_BASE_URL_KS}ksctl" "${GH_BASE_URL_CRT}host-operator" "${GH_BASE_URL_CRT}member-operator", "${GH_BASE_URL_CRT}registration-service", ${GH_BASE_URL_CRT}toolchain-e2e, ${GH_BASE_URL_CRT}toolchain-common)
```

and loop over it with :
```
for repo in "${repos[@]}"
```

Reply:
You could also separate repos for ksctl and toolchain for better readability like:
`KSCTL_REPOS=(${GH_BASE_URL_KS}ksctl)
TOOLCHAIN_REPOS=(${GH_BASE_URL_CRT}host-operator GH_MEMBER=${GH_BASE_URL_CRT}member-operator GH_REGSVC=${GH_BASE_URL_CRT}registration-service GH_E2E=${GH_BASE_URL_CRT}toolchain-e2e)
ALL_REPOS=("${KSCTL_REPOS[@]}" "${TOOLCHAIN_REPOS[@]}")`

and then loop over as francisc suggested

Reply:
i have changed it now..